### ISSUE TITLE
[JavaFX Application Thread] ERROR com.github.engatec.vdl.service.QueueItemDownloadService - WARNING: Using allow-unsafe-ext opens you up to potential attacks. Use with great care!

### ISSUE THREAD

**Issue by aibymind** [engatec/vdl#40]
I have a problem with this, when I try to download video from youtube.

Log:
```
2024-07-02 18:32:20.546 [JavaFX Application Thread] ERROR com.github.engatec.vdl.service.QueueItemDownloadService - WARNING: Using allow-unsafe-ext opens you up to potential attacks. Use with great care!
ERROR: <lambda>() got an unexpected keyword argument 'prepend'
```

How I can solve this problem?

**Comment by tom54rus** [comment]
Sorry for late response. The release fixing the issue will be created shortly.

**Comment by tom54rus** [comment]
Fixed the issue in 1.9.4
